src/main/drivers/accgyro_spi_mpu9250.c

```diff
@@ -50,47 +50,48 @@ static void mpu9250AccAndGyroInit(gyroDev_t *gyro);
 
 static bool mpuSpi9250InitDone = false;
 
-static IO_t mpuSpi9250CsPin = IO_NONE;
+#define DISABLE_MPU9250(spiCsnPin)       IOHi(spiCsnPin)
+#define ENABLE_MPU9250(spiCsnPin)        IOLo(spiCsnPin)
 
-#define DISABLE_MPU9250       IOHi(mpuSpi9250CsPin)
-#define ENABLE_MPU9250        IOLo(mpuSpi9250CsPin)
-
-void mpu9250ResetGyro(void)
+void mpu9250SpiResetGyro(void)
```

Comment:
Yes, it certainly should. I'm declaring it out of scope for this PR though, since it is a bit of a pain to do.